consent-ws/src/main/java/org/broadinstitute/consent/http/resources/ElectionReviewResource.java

```diff
@@ -64,7 +64,7 @@ public ElectionReview getElectionReviewByElectionId(@PathParam("electionId") Int
     public ElectionReview getAccessElectionReviewByReferenceId(@PathParam("electionId") Integer electionId, @QueryParam("isFinalAccess") Boolean isFinalAccess) {
         Election election = electionAPI.describeElectionById(electionId);
         Election consentElection = electionAPI.getConsentElectionByDARElectionId(election.getElectionId());
-        List<String> dataSetId = accessRequestAPI.describeDataAccessRequestFieldsById(election.getReferenceId(), Arrays.asList(DarConstants.DATASET_ID)).get(DarConstants.DATASET_ID, List.class);
+        List<Integer> dataSetId = accessRequestAPI.describeDataAccessRequestFieldsById(election.getReferenceId(), Arrays.asList(DarConstants.DATASET_ID)).get("datasetId", List.class);
```

Comment:
i think you can still use the DATASET_ID constant here

Reply:
yes, we are saving the dataset id in Data Access Request Application,  not the object id any more because it's optional now.

Reply:
i was just saying that DarConstants.DATASET_ID was changed to "datasetId" in the get call. I think you should still be able to use the constant instead of the string

Reply:
:+1: